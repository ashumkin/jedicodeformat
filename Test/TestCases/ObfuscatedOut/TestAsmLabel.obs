unit testasmlabel;interface implementation procedure test;label lbl;asm
lbl:mov eax,[eax+20h]
end;end.