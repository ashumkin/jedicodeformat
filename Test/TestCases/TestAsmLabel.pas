unit TestAsmLabel;

{ AFS 15 Jan 2006
 This unit compiles but is not semantically meaningfull
 it is test cases for the code formatting utility

  Test labels and numeric consts in asm
 }


interface

implementation


Procedure Test;
Label
  lbl;
Asm
  lbl: mov eax, [eax+20h]
End;

end.