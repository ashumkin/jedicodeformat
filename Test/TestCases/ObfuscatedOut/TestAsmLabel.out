unit testasmlabel;

interface

implementation

procedure test;
label
  lbl;
asm
         lbl:MOV EAX,[EAX+20h]
end;

end.
